.github/workflows/build_container.yml

```diff
@@ -0,0 +1,75 @@
+---
+# template source: https://github.com/bretfisher/docker-build-workflow/blob/main/templates/call-docker-build.yaml
+name: Docker Build
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
The idea is to have always a « latest » tag. It’s very useful to be able to test a new feature without waiting a new release (like today, last release was 2 month ago, but a lot of commits was done since).
For PR, it’s almost the same idea : give an easy way to directly test the proposed PR.

All these build will have specifics, logicals, tags. So pushes on main will have « main » and «latest » tags, the PR will have « pr-xxxxxxx »,  and a release with a semver name (for example 1.2.3) will have « 1 », « 1.2 » and « 1.2.3 » tags. A users can use it to access the version they want, according to their needs

Reply:
To be honest, I'm not totally confortable with the idea of building unverified code (and not sure it can be possible, because of perms).
But I remain in the idea that not having to build yourself during a review can save time.

Another solution would be with a comment. Someone with write access to the repo can comment with "/build" (for example) and it trigger a build.

[1] https://dev.to/zirkelc/trigger-github-workflow-for-comment-on-pull-request-45l2